miden/src/tests.rs

```diff
@@ -0,0 +1,170 @@
+use crate::{
+    assembly, BaseElement, FieldElement, ProgramInputs, Serializable, Trace, TraceMetadata,
+    TraceState, TraceTable,
+};
+use air::ToElements;
```

Comment:
Most tests in this module seem like old tests. I'm actually surprised they compile. maybe this file is not included at all in the project?

The new tests you've added here should probably go into the integration tests directory.

Reply:
Ya this is not new tests I added, I think I added it during merge conflicts by mistake. I'll juts remove this